spyder/widgets/mixins.py

```diff
@@ -897,15 +897,25 @@ def get_text_region(self, start_line, end_line):
         start_cursor.setPosition(end_position, mode=QTextCursor.KeepAnchor)
         return self.get_selected_text(start_cursor)
 
-    def get_text(self, position_from, position_to):
+    def get_text(self, position_from, position_to, remove_newlines=None):

```

Comment:
I think we change this to `remove_newlines=True` because it's easier to understand the code below that way.